src/MigrationVersions/content_type.yaml

```diff
@@ -476,6 +476,179 @@
+            identifier: short_description
+            type: ezrichtext
+            name:
+                eng-GB: Summary
+            description:
+                eng-GB: 'Displayed as a separate paragraph at top.'
+            required: false
+            searchable: true
+            info-collector: false
+            disable-translation: false
+            category: content
+            position: 3
+            field-settings: {  }
+            validator-configuration: {  }
+        -
+            identifier: description
+            type: ezrichtext
+            name:
+                eng-GB: Description
+            description:
+                eng-GB: 'Body of text.'
+            required: false
+            searchable: true
+            info-collector: false
+            disable-translation: false
+            category: content
+            position: 4
+            field-settings: {  }
+            validator-configuration: {  }
+        -
+            identifier: image
+            type: ezimage
+            name:
+                eng-GB: Image
+            description:
+                eng-GB: 'Image for header.'
+            required: false
+            searchable: false
+            info-collector: false
+            disable-translation: false
+            category: content
+            position: 5
+            field-settings: {  }
+            validator-configuration:
+                FileSizeValidator: { maxFileSize: null }
+        -
+            identifier: show_children
+            type: ezboolean
+            name:
+                eng-GB: 'Display sub items'
+            description:
+                eng-GB: 'If checked folder items displayed.'
+            required: false
+            searchable: false
+            info-collector: false
+            disable-translation: true
+            category: content
+            position: 6
+            default-value: true
+            field-settings: {  }
+            validator-configuration: {  }
+        -
+            identifier: logo
+            type: ezimageasset
+            name:
+                eng-GB: Logo
+            description: {  }
+            required: false
+            searchable: true
+            info-collector: false
+            disable-translation: false
+            category: content
+            position: 7
+            field-settings: {  }
+            validator-configuration: {  }
+        -
+            identifier: css
+            type: eztext
+            name:
+                eng-GB: CSS
+            description: {  }
+            required: false
+            searchable: false
+            info-collector: false
+            disable-translation: false
+            category: content
+            position: 8
+            field-settings:
+                textRows: 10
+            validator-configuration: {  }
+        -
+            identifier: js
+            type: eztext
+            name:
+                eng-GB: JS
+            description: {  }
```

Comment:
Same here. We should mention that the javascript will be used for the whole website